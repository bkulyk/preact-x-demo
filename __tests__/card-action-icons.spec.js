import React from 'react';
import CardActionIcons from '@bitchin/card-action-icons';
import { shallow } from './enzyme';

it('renders correctly, it is pretty much just a div', () => {
  expect(shallow(<CardActionIcons>-- some children go here --</CardActionIcons>))
    .toMatchSnapshot();
});