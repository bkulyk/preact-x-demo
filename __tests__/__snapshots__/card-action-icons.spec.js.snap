// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly, it is pretty much just a div 1`] = `
<div
  className="mdc-card__action-icons"
>
  -- some children go here --
</div>
`;
